modin/core/io/column_stores/parquet_dispatcher.py

```diff
@@ -614,7 +614,10 @@ def _read(cls, path, engine, columns, **kwargs):
         ParquetFile API is used. Please refer to the documentation here
         https://arrow.apache.org/docs/python/parquet.html
         """
-        if any(arg not in ("storage_options", "use_nullable_dtypes") for arg in kwargs):
+        if any(
+            arg not in ("storage_options", "use_nullable_dtypes", "dtype_backend")
```

Comment:
don't we want to default to pandas for every `.read_*` method if `dtype_backend` was specified?

Reply:
Not every function needs it. I'll go through all the functions and check https://github.com/modin-project/modin/pull/5995#issuecomment-1573727109.